compiler/tests/org/jetbrains/kotlin/code/CodeConformanceTest.kt

```diff
@@ -138,7 +138,7 @@ class CodeConformanceTest : TestCase() {
         for (sourceFile in FileUtil.findFilesByMask(SOURCES_BUNCH_FILE_PATTERN, root)) {
             if (EXCLUDED_FILES_AND_DIRS.any { FileUtil.isAncestor(it, sourceFile, false) }) continue
 
-            val matches = Regex("BUNCH (\\w+)").findAll(sourceFile.readText())
+            val matches = Regex("""BUNCH (\d+)""").findAll(sourceFile.readText())
```

Comment:
Actually it can be even better: """BUNCH \d\d\d""". But for some reason I left older behavior